authority/ssh.go

```diff
@@ -326,15 +344,15 @@ func (a *Authority) RenewSSH(ctx context.Context, oldCert *ssh.Certificate) (*ss
 	va := now.Add(-1 * backdate)
 	vb := now.Add(duration - backdate)
 
-	// Build base certificate with the key and some random values
+	// Build base certificate with the old key.
+	// Nonce and serial will be automatically generated on signing.
 	cert := &ssh.Certificate{
```

Comment:
this is technically only a certTemplate. I think we should stick to only referring to certificates as things that are already signed.

Reply:
You want me to the variable name, right?

Reply:
https://github.com/smallstep/certificates/blob/193d18ee21bd9a476222d55337569fe17ab4988d/authority/tls.go#L389

Reply:
I think I just want to be clear around the fact that this isn't a signed cert. It's more confusing when we use `GetCertificate` because it's not actually returning a signed certificate. In this case it's not as bad. But still, it would be nice to be consistent.

Reply:
Fixed in 4d375a0